### PR TITLE
Fix `!important` ast

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Fix AST of `!important`, because `important` work is identifier we should use Ident AST node here, also it allows to parse and keep how important was written

**BREAKING CHANGE:**

Yes

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_stylis/src/prefixer.rs [hunk 1]
```diff
@@ -469,17 +491,23 @@ impl VisitMut for Prefixer {
...
                 let mut value = n.value.clone();
+
                 replace_ident(&mut value, "transform", "-webkit-transform");
+
                 let name = DeclarationName::Ident(Ident {
                     span: DUMMY_SP,
                     value: "-webkit-transition".into(),
                     raw: "-webkit-transition".into(),
                 });
+                let important = match &n.important {
```
Comment (kdy1) [inline crates/swc_stylis/src/prefixer.rs:502]: I think `n.important.clone()` would be better.

File: crates/swc_stylis/src/prefixer.rs [hunk 2]
```diff
@@ -257,11 +275,15 @@ impl VisitMut for Prefixer {
                         .collect();
 
                     if n.value != new_value {
+                        let important = match &n.important {
```
Comment (kdy1) [inline crates/swc_stylis/src/prefixer.rs:278]: Same here

File: crates/swc_stylis/src/prefixer.rs [hunk 3]
```diff
@@ -209,11 +223,15 @@ impl VisitMut for Prefixer {
                                 },
                                 value: f.value.clone(),
                             });
+                            let important = match &n.important {
```
Comment (kdy1) [inline crates/swc_stylis/src/prefixer.rs:226]: Same here.

File: crates/swc_stylis/src/prefixer.rs [hunk 4]
```diff
@@ -185,11 +195,15 @@ impl VisitMut for Prefixer {
...
```
Comment (kdy1) [inline crates/swc_stylis/src/prefixer.rs:198]: Same here.

File: crates/swc_stylis/src/prefixer.rs [hunk 5]
```diff
@@ -67,12 +74,15 @@ impl VisitMut for Prefixer {
                     value: $name.into(),
                     raw: $name.into(),
                 };
-
+                let important = match &n.important {
```
Comment (kdy1) [inline crates/swc_stylis/src/prefixer.rs:77]: Same here.

File: crates/swc_stylis/src/prefixer.rs [hunk 6]
```diff
@@ -50,12 +54,15 @@ impl VisitMut for Prefixer {
                     value: $name.into(),
                     raw: $name.into(),
                 });
-
+                let important = match &n.important {
```
Comment (kdy1) [inline crates/swc_stylis/src/prefixer.rs:57]: Same here.

File: crates/swc_stylis/src/prefixer.rs [hunk 7]
```diff
@@ -34,11 +34,15 @@ impl VisitMut for Prefixer {
                     value: $name.into(),
                     raw: $name.into(),
                 });
+                let important = match &n.important {
```
Comment (kdy1) [inline crates/swc_stylis/src/prefixer.rs:37]: Same here.

**Review by kdy1 — CHANGES_REQUESTED** [review]
Thanks!

**Comment by alexander-akait** [comment]
@kdy1 Done :+1:

**Review by kdy1 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**